demos/python_demos/human_pose_estimation_demo/human_pose_estimation.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python3
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging
+import os.path as osp
+import sys
+from argparse import ArgumentParser, SUPPRESS
+from itertools import cycle
+from enum import Enum
+from time import perf_counter
+
+import cv2
+import numpy as np
+from openvino.inference_engine import IECore
+
+from human_pose_estimation_demo.model import HPEAssociativeEmbedding, HPEOpenPose
+from human_pose_estimation_demo.visualization import show_poses
+
+sys.path.append(osp.join(osp.dirname(osp.dirname(osp.abspath(__file__))), 'common'))
+import monitors
+from helpers import put_highlighted_text
+
+
+logging.basicConfig(format='[ %(levelname)s ] %(message)s', level=logging.INFO, stream=sys.stdout)
+log = logging.getLogger()
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument('-i', '--input', help='Required. Path to an image, video file or a numeric camera ID.',
+                      required=True, type=str)
+    args.add_argument('-m', '--model', help='Required. Path to an .xml file with a trained model.',
+                      required=True, type=str)
+    args.add_argument('--type', choices=('ae', 'openpose'), required=True, type=str,
```

Comment:
Please, rename this key to `-at, --architecture_type`. (see [here](https://github.com/openvinotoolkit/open_model_zoo/pull/1744/files#diff-503f779bfaf37b3dcde2b10d8fe476d15a842bfa01166968c74252a10295e8c7R47))

Reply:
Done.